data/fwupd.conf

```diff
@@ -1,5 +1,2 @@
-# use `man 5 fwupd.conf` for documentation
 [fwupd]
-DisabledPlugins=test;test_ble
-OnlyTrusted=true
-AllowEmulation=false
+# use `man 5 fwupd.conf` for documentation
```

Comment:
Reason for this change is the key file parser doesn't like a totally empty section.... 😄